types/Decimal.php

```diff
@@ -99,7 +99,7 @@ public function renderValue($value, \Doku_Renderer $R, $mode)
     public function validate($rawvalue)
     {
         $rawvalue = parent::validate($rawvalue);
-        $rawvalue = str_replace(',', '.', $rawvalue); // we accept both
+        $rawvalue = trim((float)str_replace(',', '.', $rawvalue), 0); // we accept both
```

Comment:
If my value is `1000` this will reduce it to `1`.

```
echo trim((float) '1000', 0);
1
```

 I believe casting the value to float should already do what you want. 

```
echo (float) '00004440000';
4440000
```

Reply:
I think you are right. I will change that.